Mathlib/GroupTheory/Subsemigroup/Center.lean

```diff
@@ -33,39 +33,46 @@ namespace Set
 variable (M)
 
 /-- The center of a magma. -/
-@[to_additive add_center " The center of an additive magma. "]
+@[to_additive addCenter " The center of an additive magma. "]
 def center [Mul M] : Set M :=
```

Comment:
I believe the [naming convention](https://github.com/leanprover-community/mathlib4/wiki/Porting-wiki#naming-convention) would be `Center`, `AddCenter`.

Reply:
One of these days I'll get the naming conventions right from the get-go :) Thank you!

Reply:
Hah, blind leading the blind here ... :)